core/build.gradle

```diff
@@ -9,8 +9,7 @@ dependencies {
     compile "com.googlecode.libphonenumber:carrier:$googleCarrierVersion"
 
     // Optimaize language detection
-    compile "com.salesforce.transmogrifai:language-detector:$optimaizeLangDetectorVersion"
-
+    compile "com.optimaize.languagedetector:language-detector:$optimaizeLangDetectorVersion"
```

Comment:
There was some fixes I did into Optimaize language detector. I think it was related to Guava.

Reply:
https://github.com/optimaize/language-detector/pull/74

Reply:
I was hoping to get rid of this fork by shading if necessary.

Reply:
Sure. No development is happening on that repo, so not a big deal imo.